src/server/scripts/Northrend/Ulduar/Ulduar/boss_yoggsaron.cpp

```diff
@@ -429,6 +445,18 @@ class boss_yoggsaron_sara : public CreatureScript
 
         void Reset() override
         {
+            if (!_secondPhase) // Phase 1 wipe
+            {
+                Map::PlayerList const& pList = me->GetMap()->GetPlayers();
```

Comment:
you can use instance->DoForAllPlayers()

Reply:
I like that one, didn't even know it existed.